### ISSUE TITLE
Regressive bug: Pressing enter on auto suggestion popup, goes to newline. 

### ISSUE THREAD

**Issue by nandakishore2009** [notepad-plus-plus/notepad-plus-plus#11178]
### Description of the Issue
Pressing enter on auto suggestion popup selection, goes to newline. Prior to version v8.2, doing a keyboard selection on auto suggestion popup using up/down arrow key and pressing enter will print the selected item without going to the new line.

### Steps to Reproduce the Issue
1. Do a keyboard selection on suggestion popup and press enter
<img width="144" alt="Selection" src="https://user-images.githubusercontent.com/24523188/153539076-9de0dbc2-a847-4eb3-a033-819b1e2b6b3d.png">

2. The selected content is printed, but instead of staying on the same line, its goes to the newline.
<img width="145" alt="On selection goes to new line" src="https://user-images.githubusercontent.com/24523188/153539185-dbbfe675-8981-4174-84d4-dbbd08ceb84b.png">
 
### Debug Information
Notepad++ v8.1.9.3   (64-bit)
Build time : Dec  6 2021 - 19:21:37
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Home Single Language (64-bit) 
OS Version : 1909 
Current ANSI codepage : 1252
Plugins : DSpellCheck.dll jN.dll mimeTools.dll NppConverter.dll NppExport.dll PythonScript.dll XMLTools.dll

**Comment by deebash** [comment]
Unable to reproduce the issue in version 8.3 (64-bit)

**Comment by Glothan** [comment]
Still present in 8.3 (64-bit).
Pressing "enter" do not print the suggested word and goes to newline

**Comment by deebash** [comment]
Notepad++ v8.3   (64-bit)
Build time : Feb 11 2022 - 11:53:40
Built with : GCC 11.2.0
Path : E:\deebash\opensource\notepad-plus-plus\PowerEditor\gcc\bin.x86_64\notepad++.exe
Command Line : 
Admin mode : ON
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 1909
OS Build : 18363.476
Current ANSI codepage : 1252
Plugins : none

The above debug info might help

**Comment by Glothan** [comment]
Notepad++ v8.3   (64-bit)
Build time : Feb  3 2022 - 03:39:05
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 (64-bit) 
OS Version : 2009
OS Build : 22000.493
Current ANSI codepage : 1252
Plugins : HexEditor.dll mimeTools.dll NppConverter.dll NppExport.dll XMLTools.dll

**Comment by CennoxX** [comment]
What is your Insert Selection in Auto Completion?
![grafik](https://user-images.githubusercontent.com/6975881/153564644-1555cce1-46fb-44bb-87c6-b0b61e1e3016.png)

**Comment by CennoxX** [comment]
@Glothan Select Enter in Insert Selection and it is printed again, maybe there was a change in the default values. As said this is not was the original issue was about.

**Comment by Glothan** [comment]
@CennoxX Thank you very much, now it works, so the default value was changed among the versions. Sorry if I wrote in this post, I thought it was the same issue

**Comment by nandakishore2009** [comment]
> Here it is. I confirm the suggested word is not printed ![image](https://user-images.githubusercontent.com/19431102/153567065-aa48c05f-fcd1-45e3-8ce7-3b7c51c15b75.png)

Thanks selecting **Insert Selction > Enter** worked. It seems during the upgrade to v8.3 **Insert Selction > Enter** got unticked.

**Comment by alankilborn** [comment]
> It seems during the upgrade to v8.3 Insert Selction > Enter got unticked.

No, that's a new option; when you upgraded to a version that supported the new option, the new option was defaulted different than previous behavior.
